<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CyberFleet GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="MainArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>10</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="License" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>1</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2"/>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="LoginScreen" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>1</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QGridLayout" name="gridLayout_2"/>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="PortArea" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>1</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4"/>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="TechArea" native="true"/>
       </item>
       <item>
        <widget class="QWidget" name="FactoryArea" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>1</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QGridLayout" name="gridLayout_3"/>
        </widget>
       </item>
      </layout>
      <zorder>FactoryArea</zorder>
      <zorder>License</zorder>
      <zorder>LoginScreen</zorder>
      <zorder>PortArea</zorder>
      <zorder>TechArea</zorder>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="LogBrowser">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>3</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>5</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="CommandPrompt">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>5</height>
       </size>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>916</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSortie">
    <property name="title">
     <string>Sortie</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFleet">
    <property name="title">
     <string>Fleet</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSupply">
    <property name="title">
     <string>Supply</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuRefit">
    <property name="title">
     <string>Refit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuDock">
    <property name="title">
     <string>Dock</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFactory">
    <property name="title">
     <string>Factory</string>
    </property>
    <addaction name="actionDevelop_Equipment"/>
   </widget>
   <widget class="QMenu" name="menuNaval_Base">
    <property name="title">
     <string>Naval Base</string>
    </property>
    <addaction name="actionBack_to_naval_base"/>
    <addaction name="actionLogout"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTech">
    <property name="title">
     <string>Tech</string>
    </property>
    <addaction name="actionView_Tech"/>
   </widget>
   <addaction name="menuNaval_Base"/>
   <addaction name="menuTech"/>
   <addaction name="menuSortie"/>
   <addaction name="menuFleet"/>
   <addaction name="menuSupply"/>
   <addaction name="menuRefit"/>
   <addaction name="menuDock"/>
   <addaction name="menuFactory"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDevelop_Equipment">
   <property name="text">
    <string>Develop Equipment</string>
   </property>
  </action>
  <action name="actionBack_to_naval_base">
   <property name="text">
    <string>Back to naval base</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit program</string>
   </property>
  </action>
  <action name="actionView_Tech">
   <property name="text">
    <string>View Tech</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>LogBrowser</tabstop>
  <tabstop>CommandPrompt</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>processCmd()</slot>
  <slot>parseConnectReq()</slot>
  <slot>parseRegReq()</slot>
 </slots>
</ui>
